MiniGamesBox Classic/src/main/java/plugily/projects/minigamesbox/classic/handlers/setup/items/category/MaterialMultiLocationItem.java

```diff
@@ -129,30 +144,30 @@ public void onClick(InventoryClickEvent event) {
     switch(event.getClick()) {
       case LEFT:
         Block targetBlock = event.getWhoClicked().getTargetBlock(null, 7);
-        if(targetBlock.getType() != checkMaterial) {
-          new MessageBuilder("&c&l✘ &cPlease only look at a location where already is a " + checkMaterial.toString() + " to add it as a " + name.toUpperCase() + "!").prefix().send(event.getWhoClicked());
+        if(!checkMaterials.contains(targetBlock.getType()) ) {
+          new MessageBuilder("&c&l✘ &cPlease only look at a location where already is a " + checkMaterials+ " to add it as a " + name.toUpperCase() + "!").prefix().send(event.getWhoClicked());
```

Comment:
could return unreadable long message for user, getting the "clicked" type would be more accurate